### PR TITLE
fix(chat): select folder on rename & create (Issue #2022, #2421)

### PR DESCRIPTION
**Pull request by Gimir**
**Description:**

- select temporary folder after create
- select temporary folder after rename

Issues:

- Issue #2022
- Issue #2421 

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Gimir** [comment]
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/546051)

**Comment by Gimir** [comment]
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/546058)

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2768.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2768.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Chat/ChangePathDialog.tsx [hunk 1]
```diff
@@ -172,7 +170,7 @@ export const ChangePathDialog = ({
     (parentFolderId = rootFolderId) => {
       const folderName = getNextDefaultName(
         t(DEFAULT_FOLDER_NAME),
-        folders,
+        folders.filter((f) => f.publishedWithMe || f.temporary),
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/ChangePathDialog.tsx]: `ChangePathDialog` can be used not only for `PublishWizard`
```suggestion
        folders.filter((f) => f.folderId === parentFolderId),
```

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2768.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**